Mathlib/Algebra/Group/Aut.lean

```diff
@@ -255,4 +265,14 @@ theorem conj_symm_apply [AddGroup G] (g h : G) : (conj g).symm h = -g + h + g :=
 theorem conj_inv_apply [AddGroup G] (g h : G) : (Additive.toMul (conj g))⁻¹ h = -g + h + g :=
   rfl
 
+/-- Isomorphic additive groups have isomorphic automorphism groups. -/
+@[simps]
+def congr [AddGroup G] {H : Type*} [AddGroup H] (ϕ : G ≃+ H) :
+    AddAut G ≃* AddAut H where
```

Comment:
Why can't we get this with `@[to_additive]`?

Reply:
I didn't check actually, but I assumed not since I thought that `to_additive` would try to turn `≃*` into `≃+` (but we want both the multiplicative and additive versions to use `≃*`).

Reply:
Ah, that should have been obvious, thanks!